pkg/virt-controller/services/template.go

```diff
@@ -78,6 +78,12 @@ func (t *templateService) RenderLaunchManifest(vm *v1.VM) (*kubev1.Pod, error) {
 		},
 	}
 
+	if vm.Spec.NodeAffinity != nil {
```

Comment:
We also add an affinity to the underlying pod, if we do a migration. Could you add a test, to make sure, that when rendering the migration Pod, we don't override such a pre-set affinity?